CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+## How to contribute to `gleam`
+
+Thank you for taking the time to contribute to `gleam`!
+
+### Join
+
+The `gleam` community follows the [astropy code of conduct](https://www.astropy.org/code_of_conduct.html).
+If you have read it, then you know that the most important contribution is to treat others with respect.
+
+### Share
+
+A great way to contribute to `gleam` is to use it in your projects and then talk about it. Share
+your experience with other people who might be interested in using `gleam` and direct them to the
+[GitHub repository page](https://github.com/multiwavelength/gleam).
+
+### Suggest
+
+Your experience is important, so any time you feel like it could be improved, please open an
```

Comment:
Your experience using `gleam' is important, so any time you feel like it could be improved (e.g. you find a bug or have a feature request), ...

Reply:
I adapted your suggestion and I used less technical terms for bugs and feature requests.